crates/iceberg/src/spec/snapshot.rs

```diff
@@ -72,7 +73,7 @@ pub struct Snapshot {
     sequence_number: i64,
     /// A timestamp when the snapshot was created, used for garbage
     /// collection and table inspection
-    timestamp_ms: i64,
+    timestamp_ms: Timestamp,
```

Comment:
We used to have a discussion about this.

https://github.com/apache/iceberg-rust/pull/20/files#diff-0ab4b2d8c9bcfbb43cb0a06b6ef3015098edf660cbeebb5f196a55eba0326b83

The general idea is to **use long/int representation**.

We just convert them to chrono types as public API.

---

*By the way, I still believe that directly storing a `DateTime<Utc>` is a good idea. However, I don't want to reopen a discussion on a topic we've already reached consensus on.*

Reply:
> We used to have a discussion about this.
> 
> #20 (comment)
> 
> The general idea is to use long/int representation.
> 
> We just convert them to chrono types as public API.

+1. We have reached consensus, so we should not change the internal field representation without a strong reason. What I meant was also to return chrono types as public api.

Reply:
Was not aware of any of this history, thanks!

> return chrono types as public api.

If I'm understanding correctly, it sounds like we only want to modify APIs marked as `pub`, `pub(crate)`, `pub(super)`, etc. so that they return a `DateTime<Utc>` instead of `i64`.

If so,  the latest [commit](https://github.com/apache/iceberg-rust/pull/94/commits/9521da820f14fc0383c65128b368a5764ed028b5) should address these concerns.